### ISSUE TITLE
A.3.24.1100. - eConsent custom message

### ISSUE THREAD

**Issue by aldefouw** [4bbakers/redcap_rsvc#24]
Feature: [A.3.24.1100. - eConsent custom message](https://github.com/4bbakers/redcap_rsvc/tree/v14.7.0/./Feature%20Tests/A/e-Consent%20framework_24/A.3.24.1100.%20-%20eConsent%20custom%20message.feature)

**Comment by ellisthomas** [comment]
@mmcev106  I think this one is failing at this step. And I select "Participant Consent" from the dialogue box labeled "Enable e-Consent for a Survey". It states that it is an undefined step which we talked about with these dialog box actions. I also had to move the XML file labled '24EConsentNoSetup.xml' into the cdisc_file under redcap_cypress.

**Comment by mmcev106** [comment]
@ellisthomas, I think we got this one squared away on our call, but let me know if anything else comes up!

**Comment by ellisthomas** [comment]
@mmcev106  @aldefouw 
![Image](https://github.com/user-attachments/assets/3f57e640-0b4c-40c3-8f30-64526ea6aaa5)

The custom text is being put into the wrong field this not showing the custom message in later steps. Then I should see a field labeled "Enable e-Consent"
          And I should see "Primary settings"
          And I should see "My custom message"

**Comment by mmcev106** [comment]
@ellisthomas, am I correct that the following step is the one that is putting the text into the wrong field?
```
When I enter "My custom message" into the input field labeled "Custom message for e-Consent Framework settings"
````

**Comment by ellisthomas** [comment]
@mmcev106 That is correct. This step is where the text "My custom message" is put into the wrong field.

**Comment by mmcev106** [comment]
@aldefouw, do you have any guidance here?  No worries if not.  I'm happy to dive into it.

**Comment by aldefouw** [comment]
@aldefouw - I'll take a peek and let you know if I see anything obvious.

**Comment by aldefouw** [comment]
@mmcev106  & @ellisthomas - 

This one had some gotchas but was short so therefore easy to review.  The primary issue you were referencing was related to the fact that the field we wanted to target was a textarea and not an input field.

Surprisingly, it was missing the login as "Test_Admin" for some reason so I added that and cleaned it up a bit.

I pushed a few changes to my branch - see what you think about the changes I am proposing: 
https://github.com/aldefouw/redcap_rsvc/commit/a2f17e20cfe50a33657b5f88920de59771fb1338

If you agree @mmcev106 feel free to merge; bonus - then you're off the hook @ellisthomas - for this one at least!
